packages/vant/src/calendar/CalendarMonth.tsx

```diff
@@ -87,7 +88,7 @@ export default defineComponent({
 
     const shouldRender = computed(() => visible.value || !props.lazyRender);
 
-    const getTitle = () => title.value;
+    const getTitle = () => toRef(title, 'value');
```

Comment:
Is this a breaking change? The `title` param of `month-show` event is changed.

The previous type was string, but now the type is Ref<string>

Reply:
this is a  breaking change. the `title` should change responsively with `locale` changes.

The `title` type didn't change, that was my mistake.